src/antenna.h

```diff
@@ -21,8 +21,17 @@
 // Currently 30 minutes
 #define COMMS_DELAY_DEF_S (30 * 60)
 
+#define ANT_REL_A       6
+#define ANT_REL_B       7
```

Comment:
Define these as PC6 and PC7 to be consistent with how we named all the other GPIO constants